src/StyleSheet.js

```diff
@@ -30,22 +29,26 @@ export default class StyleSheet {
     this.rules = Object.create(null)
     // Used to ensure correct rules order.
     this.rulesIndex = []
-    this.classes = Object.create(null)
+    this.classes = this.options.classes = Object.create(null)
```

Comment:
Another option: create classes object in RulesContainer constructor and then reference it in StyleSheet:

```js
this.rules = new RulesContainer(this.options)
this.classes = this.rules.classes
```